discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "inline",
+            "tutorials": "yes",
+            "examples": "yes",
+            "eventsAsync": "threaded",
+            "apiCallsAsync": "no",
```

Comment:
Whether calls to Discord's REST API are asynchronous in any way. Can be `coroutine`, `threaded`, or `no`.